src/components/Upload.vue

```diff
@@ -1,17 +1,20 @@
 <template lang="pug">
-.upload-wrapper
-  slot(name="toolbar")
-    v-toolbar(dense, dark, color="success")
```

Comment:
If we are removing the toolbar, we may want to remove the `closeable` attribute and corresponding button since it may now be out of scope for this component.

Reply:
+1 done.